Samples/UIWidgetsSamples_2019_4/Assets/Script/CountTest.cs

```diff
@@ -14,11 +14,6 @@ namespace UIWidgetsSample
 {
     public class CountTest : UIWidgetsPanel
     {
-        protected void OnEnable()
```

Comment:
The code here would produce a compilation warning because it would hide the base class virtual method. Since there is no new behavior defined here, I think we can just remove the method.